### PR TITLE
Added algorithms in Go

### PR DESCRIPTION
**Pull request by singh-shreya6**
#501 Adding Algorithms in Go
Bubble Sort
Sieve of Eratosthenes
Quick Sort

### CODE REVIEW

**Comment by jainaman224** [comment]
Please refer the issue like `#issue_number`.

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+func main(){
+    var n int= 30
+    fmt.Printf("Following are the prime numbers smaller than or equal to %d \n", n)
+    SieveOfEratosthenes(n)
+}
+
+/*
+Following are the prime numbers smaller than or equal to 30
+2 3 5 7 11 13 17 19 23 29
+*/
```
Comment (jainaman224) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.go]: Add a new line at the end of file.

**Comment by singh-shreya6** [comment]
@jainaman224 changes done. Please review.

**Comment by singh-shreya6** [comment]
@jainaman224 #501 
Added Quick Sort in Go

**Pull request merged**